src/data_helper.py

```diff
@@ -105,7 +104,14 @@ def get_train_generator(self, batch_size):
                     # Augment the image `img` here
 
                     # Convert to RGB and normalize
-                    img_array = np.asarray(img.convert("RGB"), dtype=np.float32) / 255
+                    img_array = np.asarray(img.convert("RGB"), dtype=np.float32)
+
+                    img_array = img_array[:, :, ::-1]
+                    # Zero-center by mean pixel
+                    img_array[:, :, 0] -= 103.939
+                    img_array[:, :, 1] -= 116.779
+                    img_array[:, :, 2] -= 123.68
```

Comment:
Oh nice! :octocat:  Does it means that we'll use the mean for **all** models from now? (Vgg, Resnet...)